### DISCUSSION TITLE
✅ ♿ Elementor Pro Slides Widget add Alt for image to comply with accessibility WCAG Conformance

### DISCUSSION THREAD

**Discussion by SusanStren** [elementor/elementor#23543] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

SiteImprove is flagging "Image missing a text alternative" on every page we are using the Elementor Pro slides widget as a level A WCAG conformance  issue. 

### Describe the solution you'd like

I see that Elementor has fixed the same issue in a recent plugin update for the Carousel widget #14052 and I am hoping they can add image alt attribute for the slides widget. Or give me some kind of work around to call the alt text to the images for this widget.

**Comment by rami-elementor** [comment]
Planed for v3.32.0

**Comment by nicholaszein** [comment]
Hello there @SusanStren !

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Editor v3.32.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
